cloudkms/keysource.go

```diff
@@ -0,0 +1,130 @@
+package cloudkms //import "go.mozilla.org/sops/cloudkms"
+
+import (
+	"encoding/base64"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+
+	cloudkms "google.golang.org/api/cloudkms/v1"
+)
+
+// this needs to be a global var for unit tests to work (mockKMS redefines
+// it in keysource_test.go)
+// var kmsSvc kmsiface.KMSAPI
```

Comment:
Leftover from copying from AWS KMS. This should probably have tests like the AWS KMS keysource does, so we'll probably need something like this for the GCP version.